tableau_utilities/general/cli_styling.py

```diff
@@ -39,7 +39,7 @@ class Symbol:
     arrow_l: str = '←'
     line: str = '━'
     success: str = '✅'
-    warning: str = '⚠'
+    warning: str = '⚠️'
```

Comment:
@JustinGrilli  I tried to get a yellow triangle to show for this but still just seeing plain text.  I'll leave it yo you if you want to do that.  I can't figure it out.  

We can just ignore this too.


<img width="261" alt="image" src="https://user-images.githubusercontent.com/9727812/225920689-8b8634f8-9309-4b19-991e-f6a26957a965.png">